src/vs/workbench/contrib/notebook/browser/view/cellParts/codeCell.ts

```diff
@@ -61,6 +61,7 @@ export class CodeCell extends Disposable {
 		// this.viewCell.layoutInfo.editorHeight or estimation when this.viewCell.layoutInfo.editorHeight === 0
 		const editorHeight = this.calculateInitEditorHeight();
 		this.initializeEditor(editorHeight);
+		this._renderedInputCollapseState = false; // editor is always expanded initially
```

Comment:
does this apply to IW? inputs from .py files are collapsed initially

Reply:
@amunger `this.initializeEditor` creates the editor related components (DOM nodes) so the initial state of `this._renderedInputCollapseState` should be `false`. Then the Jupyter will ask the notebook to collapse the cell, which will be handled separately. Thus there is no impact on how IW works.